config/tokens.ts

```diff
@@ -1,5 +1,4 @@
 import { TOKENS, CHAINS } from '@lido-sdk/constants';
-import findKey from 'lodash/findKey';
 
 export const TOKENS_BY_NETWORK: {
```

Comment:
Why can't you use `TOKENS_BY_NETWORK` and `getTokenAddress` from @lido-sdk/constants?

Reply:
Basically I will remove lido-sdk/constants in the meanwhile, because it's deprecated, but also it's just a refactoring of existing code 🤔

Reply:
It is not deprecated and has up-to-date addresses of contracts and tokens. And now it is kept up to date for the stake widget

Reply:
The whole `lido-sdk` is deprecated since there is no maintainer, and no clear purpose of these packages, there are only fixes as you mentioned.

Reply:
As long as there is no clear alternative - this package is not deprecated. And it's kept up to date